.github/workflows/frontend-ci-cd.yaml

```
@@ -0,0 +1,83 @@
+name: frontend-ci-cd
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)